tests/cypress/tests/server/select_props_test.ts

```diff
@@ -115,7 +115,6 @@ describe('select row', () => {
             expectCellSelection([1, 2, 3], [3001, 3002, 3003], [0, 1], [0, 1], [2, 1], [2, 1]);
 
             // shrink the selection
-            DOM.focused.type(Key.Shift, { release: false });
```

Comment:
Cypress `3.6.x` API changed in subtle ways, doing two consecutive non-release shift key presses is actually two presses, needs to be removed for the selection to behave as expected.